### DISCUSSION TITLE
제페토 안에서 버튼을 통해 웹페이지로 이동하고 싶습니다.

### DISCUSSION THREAD

**Discussion by whalsrl2008** [naverz/zepeto-studio-kor#450] [5. Q&A]
![이미지 2022  3  2  오전 11 55](https://user-images.githubusercontent.com/76557090/156286173-9be099fb-0531-4c66-ac44-3ae264570709.jpg)
![이미지 2022  3  2  오전 11 57](https://user-images.githubusercontent.com/76557090/156286288-4894b7c9-82c8-4d51-a4d8-daec1f0228a2.jpg)
제페토 스튜디오의 UI 이벤트 사례를 참고해서 작성했는데, 에러가 발생합니다. 해결 가능한 방법이 있을까요?
+a 가능하다면, 타 브라우저(Chrome, Safari 등)를 통하지 않고, 제페토에서 네이티브로 웹 페이지를 열고 싶은데 해당 방법도 있을까요...?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 제페토 월드 심사 가이드 2.5.5에 의해 웹페이지 오픈은 지원하지 않습니다. 
참고 부탁 드립니다.🙏

https://studio.zepeto.me/kr/guides/world-guidelines

**Reply by whalsrl2008** [reply]
아하 ㅠㅠㅠ 감사합니다
